api/index.py

```diff
@@ -14,4 +15,5 @@ def nlp_analyze():
 
 
 if __name__ == '__main__':
+    LoggerFactory.get_logger(__name__).info("API started")
```

Comment:
What happens when we run unit tests ? Usually, we don't want unit tests to log, is there already something for this ?

Reply:
Logger is set by config file which has the handlers set to none by default.